corehq/apps/users/models.py

```diff
@@ -237,6 +238,9 @@ class HqPermissions(DocumentSchema):
     edit_user_tableau_config = BooleanProperty(default=False)
     view_user_tableau_config = BooleanProperty(default=False)
 
+    edit_all_user_profiles = BooleanProperty(default=True)
```

Comment:
I agree the default does not affect existing roles. I believe this permission is a subpermission of `edit_web_users ` right? (As of now, only roles with `edit_web_users` permission can edit the profile of other web users). 

In the short term, users will likely expect that enabling edit_web_users will automatically include the ability to edit profiles. However, in the long term, I think permissions should be explicitly granted. This approach errs on the side of restriction, ensuring that access is only given to users who explicitly should have it. I wonder if we should get SaaS's thoughts since this is a GA change?

Reply:
Ah ok looks like it was already discussed, the decision ya'll landed on is fine by me